src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -886,6 +892,47 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly nonQueriedMolecularData = remoteData<NumericGeneMolecularData[]>({
```

Comment:
we need a comment to explain why we need "nonQueried" data.

Reply:
Sure!